examples/selfhosted/profile/wolfssl_th_api/th_ecdsa.c

```diff
@@ -104,8 +105,17 @@ th_ecdsa_sign(void     *p_context,
     {
         case ECC_SECP256R1:
         case ECC_SECP384R1:
-            CHK1(wc_ecc_sign_hash(
-                p_msg, msglen, p_sig, p_siglen, &(c->rng), &(c->key.ecc)));
+            CHK1(wc_SignatureGenerate(
```

Comment:
wc_SignatureGenerateHash should be used, only the sign operation should be performed, not the hashing.